docs/trace/enriching-activity-processor/README.md

```diff
@@ -0,0 +1,113 @@
+# EnrichingActivityProcessor
+
+Sometimes the built-in instrumentation doesn't add enough data or you want to
+augment the data it provides with contextual information your application or
+library has available. For these cases OpenTelemetry-dotnet provides an
+`EnrichingActivityProcessor` and `EnrichmentScope` API.
+
+1. To enrich your spans first add the `EnrichingActivityProcessor` to your
```

Comment:
I guess we want to have a consistent usage of span/activity across the docs/APIs/comments?

Reply:
It is a bit of a strange situation we've put ourselves in. Anyone using Activity API will want to see Activity. Anyone using OpenTelemetry API will want to see Span. I'll switch that one to Activity since the rest of the doc uses Activity terms, but I'm not sure which way it should really be.

Reply:
Yeah, it is a hard naming problem.
I thought about this while I was writing the other docs, given .NET has `Span` which is totally different concept, I think it gives us perfect excuse to avoid using `span` in our docs (except for the API conceptual doc where we elaborate the differences).